internal/jobqueue/fsjobqueue/fsjobqueue.go

```diff
@@ -56,6 +56,15 @@ type fsJobQueue struct {
 	// reported as done.
 	jobIdByToken map[uuid.UUID]uuid.UUID
 	heartbeats   map[uuid.UUID]time.Time // token -> heartbeat
+
+	workerIDByToken map[uuid.UUID]uuid.UUID // token -> worker id
```

Comment:
Nitpick: Feel free to ignore this since it might be a pain to implement. But I'm wondering if we could/should start aliasing the `uuids` here because it gets a bit confusing and I had to read over this a few times.

Reply:
Do you think it's better now? There's a whole bunch of casts going on but ig the type fsjobqueue struct is a bit more self-explanatory now.

Reply:
Sorry just saw this comment. Yeah I agree its a bit of a trade-off, but I think its easier to parse.